{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -61,6 +61,9 @@ APP_DEFAULT_SECURE_HEADERS = {
             "blob:",            # for pdf preview
             # Add your own policies here (e.g. analytics)
         ],
+        'script-src': [
```

Comment:
could you please add a short explanation to the code what this is for?
my idea is that users who use this cookiecutter template can check in the configuration what the different settings are for